.github/workflows/main.yaml

```diff
@@ -98,11 +98,9 @@ jobs:
           echo "check_suite_id=${{ github.event.check_run.check_suite.id }}" >> $GITHUB_ENV
         else
           echo "Retrieving failed check runs for check suite ID: ${{ github.event.check_suite.id }}"
-          gh api \
-            -H "Accept: application/vnd.github+json" \
-            -H "X-GitHub-Api-Version: 2022-11-28" \
-            /repos/${{ github.repository }}/check-suites/${{ github.event.check_suite.id }}/check-runs \
-            | jq '.check_runs[] | select(.conclusion == "failure")' > failed_check_runs.json
+          gh api -X GET \
+            "repos/${{ github.repository }}/check-suites/34638332622/check-runs?per_page=100&page=1" \
```

Comment:
Did you hardcode the checksuite value on purpose?

Reply:
oups no